app/lib/graphql-api.js

```diff
@@ -0,0 +1,35 @@
+class Api {
```

Comment:
Really good work with this `Api` class that helps create GET requests for our GraphQL API. Not sure if it would help, but as our API calls grow in complexity, we could use a library (called ApolloJS) that helps formulate GraphQL requests (as the requests are obviously more hierarchical than linear REST API queries).

Reply:
Yes, I remember you mentioning this before. I've been reading up on Apollo Client (https://www.apollographql.com/docs/react/). Is that the same thing?

Reply:
That's the one!